### PR TITLE
Update dependencies to non-staging 3.0.0 versions.

### PR DESCRIPTION
**Pull request by kevinlind**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/app/build.gradle.kts [hunk 1]
```diff
@@ -58,18 +58,11 @@ android {
...
-    }
-    implementation("com.adobe.marketing.mobile:edge:3.0.0-SNAPSHOT") {
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-        exclude(group = "com.adobe.marketing.mobile", module = "edgeidentity")
-    }
-    implementation("com.adobe.marketing.mobile:edgeidentity:3.0.0-SNAPSHOT") {
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-    }
+
+    implementation("com.adobe.marketing.mobile:core:3.0.0")
```
Comment (emdobrin) [inline code/app/build.gradle.kts:62]: should we use mavenCoreVersion instead, similar to the other extensions

File: code/gradle.properties [hunk 2]
```diff
@@ -34,4 +34,4 @@ mavenRepoName=AdobeMobileEdgeBridgeSdk
 mavenRepoDescription=Adobe Experience Platform Edge Bridge extension for the Adobe Experience Platform Mobile SDK
 
 # Mobile Core dependency version
```
Comment (emdobrin) [inline code/gradle.properties:36]: could we clean this file up similar to the other extensions - e.g. https://github.com/adobe/aepsdk-edgeidentity-android/blob/staging/code/gradle.properties

**Review by emdobrin — COMMENTED** [review]
can you also include the readme/doc changes for test app run?

**Review by cacheung — COMMENTED** [review]
Can you also update the readme for this sentence
To open and run the project, open the code/settings.gradle.kt file in Android Studio

**Review by cacheung — APPROVED** [review]

**Review by cacheung — CHANGES_REQUESTED** [review]
Please update the readme.

**Review by emdobrin — APPROVED** [review]

**Review by cacheung — APPROVED** [review]

**Pull request merged**